.devcontainer/README.md

```diff
@@ -0,0 +1,30 @@
+# Envoy Dev Container (experimental)
+
+This directory contains some experimental tools for Envoy Development in [VSCode Remote - Containers](https://code.visualstudio.com/docs/remote/containers).
+
+## How to use
+
+Open with VSCode with the Container extension installed. Follow the [official guide](https://code.visualstudio.com/docs/remote/containers) to open this
```

Comment:
Can you be a bit more precise in wording here. I think it's the `Remote - Containers` extension.

Reply:
That's in L3?